extensions/ql-vscode/src/query-history.ts

```diff
@@ -950,15 +975,14 @@ export class QueryHistoryManager extends DisposableObject {
       return;
     }
 
-    // TODO(angelapwen): Stream the file in. 
-    void fs.readFile(finalSingleItem.jsonEvalLogSummaryLocation, async (err, buffer) => {
-      if (err) {
-        throw new Error(`Could not read evaluator log summary JSON file to generate viewer data at ${finalSingleItem.jsonEvalLogSummaryLocation}.`);
-      }
-      const evalLogData: EvalLogData[] = parseViewerData(buffer.toString());
+    // TODO(angelapwen): Stream the file in.
+    try {
+      const evalLogData: EvalLogData[] = await parseViewerData(finalSingleItem.jsonEvalLogSummaryLocation);
```

Comment:
This changes from a callback to awaiting processing of the summary log. Which means that `handleShowEvalLogViewer` won't complete until the log is fully processed. That seems ok to me, but @angelapwen should look at this since she originally implemented this.

Reply:
I _think_ this seems fine; it makes sense that the user would expect the command to complete once the log is processed.